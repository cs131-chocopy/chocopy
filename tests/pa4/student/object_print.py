a: object = "191abc"
b: object = 114514
c: str = "cstr"
d: int = 1919
e: bool = True
f: object = False

def fun():
    a: object = "191abc"
    b: object = 114514
    c: str = "cstr"
    d: int = 1919
    e: bool = True
    f: object = False
    print(a)
    print(b)
    print(c)
    print(d)
    print(e)
    print(f)

print(a)
print(b)
print(c)
print(d)
print(e)
print(f)

fun()
