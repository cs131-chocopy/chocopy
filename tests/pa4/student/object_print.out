191abc
114514
cstr
1919
True
False
191abc
114514
cstr
1919
True
False
